### ISSUE TITLE
[Bug] YouTube Video Issue

### ISSUE THREAD

**Issue by JuicyCologne** [Floorp-Projects/Floorp#655]
![Screenshot (393)](https://github.com/Floorp-Projects/Floorp/assets/154871924/05a85330-e0fa-4322-afc5-b27efaecdfe8)
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

So usually when I would browse(specifically YouTube) when I open a video, sometimes when I "Open link in a new tab" or just open it regularly, it would freeze the tab while the video plays and it would cover the ambient part of the video(Ambient Mode) with a black background and then the whole thing would refresh itself and back to normal and either play the video from start or continue.

### To Reproduce

Just open YouTube, click any video or "Open link in a new tab" and most of the time, should not cause the bug, but then suddenly it will cause the bug and you just sit there and watch as it fixes itself.

### Expected behavior

I just want to watch YouTube Content undisturbed:<

### OS Info

Windows 11 23H2 OS Build 22631.2792

### Floorp Version

Version 11.7.1 (64-bit) | Firefox 115.6.0

### Additional context

I just recently used Floorp, I love it for being Private and Fast

**Comment by JesusKwaist** [comment]
![Screenshot (393)](https://github.com/Floorp-Projects/Floorp/assets/154871924/cc47996c-31af-45a6-b92b-af1b5dcce075)

**Comment by regularenthropy** [comment]
Strange behavior. What add-ons do you have installed?

**Comment by surapunoyousei** [comment]
Please write issue name

**Comment by surapunoyousei** [comment]
Please write issue name

**Comment by JesusKwaist** [comment]
> Strange behavior. What add-ons do you have installed?

TWP Translate Web Pages
Return YouTube Dislike(Just installed it, problem persisted before it was in Floorp)
Search By Image(Same as Return YouTube Dislike)

**Comment by JesusKwaist** [comment]
> Please write issue name

Sorry about that, didnt notice it, added the name as requested

**Comment by surapunoyousei** [comment]
> > Strange behavior. What add-ons do you have installed?
> 
> TWP Translate Web Pages Return YouTube Dislike(Just installed it, problem persisted before it was in Floorp) Search By Image(Same as Return YouTube Dislike)

Ah... TWP makes a bug... Occasionally.

**Comment by JesusKwaist** [comment]
![image](https://github.com/Floorp-Projects/Floorp/assets/154871924/74a2b0cd-0f09-43d1-8326-ae9a5a77c4d7)

**Comment by surapunoyousei** [comment]
Ah. It's not Floorp Problem. Occurred by YouTube.

**Comment by JesusKwaist** [comment]
> Ah. It's not Floorp Problem. Occurred by YouTube.

Is there a fix or anything?

**Comment by surapunoyousei** [comment]
Rebooting Floorp is best way.

**Comment by surapunoyousei** [comment]
To be honest, this also occurs in Firefox

**Comment by JesusKwaist** [comment]
> Rebooting Floorp is best way.

I have done that every time, the problem either does not go away or does go away but comes back afterwards, clearing cache and cookies seem to temporarily work but it is annoying.

**Comment by surapunoyousei** [comment]
Yes. So it needs to be repeated.

Google often adds code that causes problems in Firefox, and there is nothing we can do about it.

You can decide to use Chromium in favor of exclusivity, or you can continue to use the Firefox family.

I have seen the effects of YouTube's malicious changes in Bugzilla dozens of times, and there is nothing I can do about it.

**Comment by JesusKwaist** [comment]
> Yes. So it needs to be repeated.
> 
> Google often adds code that causes problems in Firefox, and there is nothing we can do about it.
> 
> You can decide to use Chromium in favor of exclusivity, or you can continue to use the Firefox family.
> 
> I have seen the effects of YouTube's malicious changes at Bugzilla comment for some times, and there is nothing I can do about it.

Damn, well thank you for explaining things clearly to me.

**Comment by surapunoyousei** [comment]
My knowledge of browser development is limited as Mozilla continues to deal with it better.

Sorry.

**Comment by JuicyCologne** [comment]
> My knowledge of browser development is limited as Mozilla continues to deal with it better.
> 
> Sorry.

I have found some possible fixes:

-Go to Privacy and Security and set it to Standard(This is on by default but when I rechecked it was set to enhanced)
-Set "Do Not Track" Only when Floorp is set to block known trackers
-Disable Ambient Mode
-Disabled Hardware Acceleration 

PS: These fixes may vary, some may make performance worst(Hardware Acceleration Disabled) and to some, the problem still persisting. I've done everything but the 4th Option and so far, it rarely happens now, it may still comeback but not as occasional as it was back then. 

Hope this may help anyone experiencing the same issue!
